.github/workflows/test.yml

```
@@ -30,59 +30,30 @@ concurrency:
 permissions:
   contents: read
 jobs:
-  docker-targets:
-    name: Docker targets
-    runs-on: ubuntu-latest
-    outputs:
-      targets: ${{ steps.detect-targets.outputs.targets }}
-    steps:
-      - name: Detect targets
-        id: detect-targets
-        run: |
-          echo "targets<<JSON" >> "$GITHUB_OUTPUT"
-          echo "[" >> "$GITHUB_OUTPUT"
-          cat <<JSON >> "$GITHUB_OUTPUT"
-          {
-            "arch-label": "AMD64",
-            "arch": "amd64",
-            "go": "1.22",
-            "runs-on": "ubuntu-latest"
-          },
-          {
-            "arch-label": "AMD64",
-            "arch": "amd64",
-            "go": "1.23",
-            "runs-on": "ubuntu-latest"
-          }
-          JSON
-          if [ "$GITHUB_REPOSITORY_OWNER" = "apache" ]; then
-            echo "," >> "$GITHUB_OUTPUT"
-            cat <<JSON >> "$GITHUB_OUTPUT"
-          {
-            "arch-label": "ARM64",
-            "arch": "arm64v8",
-            "go": "1.22",
-            "runs-on": ["self-hosted", "arm", "linux"]
-          },
-          {
-            "arch-label": "ARM64",
-            "arch": "arm64v8",
-            "go": "1.23",
-            "runs-on": ["self-hosted", "arm", "linux"]
-          }
-          JSON
-          fi
-          echo "]" >> "$GITHUB_OUTPUT"
-          echo "JSON" >> "$GITHUB_OUTPUT"
   docker:
     name: ${{ matrix.arch-label }} Debian 12 Go ${{ matrix.go }}
-    needs: docker-targets
     runs-on: ${{ matrix.runs-on }}
     timeout-minutes: 30
     strategy:
       fail-fast: false
       matrix:
-        include: ${{ fromJson(needs.docker-targets.outputs.targets) }}
+        include:
+          - arch: amd64
+            go: 1.22
+            runs-on: ubuntu-latest
+            arch-label: AMD64
```

Could you keep these keys in alphabetical order?

```suggestion
          - arch-label: AMD64
            arch: amd64
            go: 1.22
            runs-on: ubuntu-latest
```

---

Done